pkg/api/types.go

```diff
@@ -1359,6 +1360,11 @@ type ProjectDirectoryImageBuildInputs struct {
 	// that will populate the build context for the Dockerfile or
 	// alter the input image for a multi-stage build.
 	Inputs map[string]ImageBuildInputs `json:"inputs,omitempty"`
+
+	// BuildArgs contains build arguments that will be resolved in the Dockerfile.
+	// See https://docs.docker.com/engine/reference/builder/#/arg for more details.
+	// Currently only value is supported.
+	BuildArgs []corev1.EnvVar `json:"build_args,omitempty"`
```

Comment:
Use a type that exposes the options we *actually* use, not some superset that misleads users and we need to validate, please.

Reply:
Updated.